diff --git a/index.js b/index.js
index 89f1b2cb86fad204b0493da3b8a3d5ed28937260..945f4cff27ed501fca75e269dfd7172e74c9c955 100644
--- a/index.js
+++ b/index.js
@@ -62,6 +62,11 @@ function supportsHyperlink(stream) {
 		return false;
 	}
 
+	// Cursor supports hyperlinks
+  if ("CURSOR_TRACE_ID" in env) {
+    return true;
+  }
+
 	if ('TERM_PROGRAM' in env) {
 		const version = parseVersion(env.TERM_PROGRAM_VERSION);
 
