diff --git a/dist/rollup.js b/dist/rollup.js
index cc82363277fe849c9460937773d9bc8b5c86beb6..ab2343b5a595f1a213de18f3d40efae15c440a4d 100644
--- a/dist/rollup.js
+++ b/dist/rollup.js
@@ -6791,7 +6791,7 @@ var getRollupConfig = async (options) => {
           tsconfig: options.tsconfig,
           compilerOptions: {
             ...compilerOptions,
-            baseUrl: compilerOptions.baseUrl || ".",
+            baseUrl: compilerOptions.baseUrl || undefined,
             // Ensure ".d.ts" modules are generated
             declaration: true,
             // Skip ".js" generation
